uwdata/lyra
# Scales inspector

**Description of the problem this pull request fixes**
- Re styles the scale sidebar
- shows the inspector on clicking
- unselects any active marks
- passes scale data to the inspector (but form is not there yet, just needs someone to set that up)

**This PR includes:**

**Steps to functionally test this:**
- add a mark
- attach data to it
- scale should show up in sidebar
- click on scale
- inspector placeholder should show up for scale with information about the scale
- click on mark - mark inspector should show
